airflow/providers/microsoft/azure/operators/data_factory.py

```diff
@@ -65,8 +66,20 @@ def get_link(
             extras, "resource_group_name"
         )
         factory_name = operator.factory_name or get_field(extras, "factory_name")  # type: ignore
+
+        azure_cloud: AzureCloud = AzureCloud.AzureGlobal
+
+        try:
+            operator = get_field(extras, "cloudOperator", strict=True)
+            if operator == "AzureChina":
+                azure_cloud = AzureCloud.AzureChinaCloud
+            elif operator == "AzureUS":
+                azure_cloud = AzureCloud.AzureUSGovernment
+        except KeyError:
+            pass
```

Comment:
Is this a good idea?
Any new region added by Azure will require code change from Airflow.

Reply:
@kzmain not sure why you marked this as resolved. I don't see you addressed this question